Cargo.toml

```diff
@@ -29,13 +29,15 @@ keywords = ["gamedev", "cgmath", "collision"]
 [lib]
 name = "collision"
 
-[features]
-eders = ["serde", "serde_derive", "cgmath/serde"]
-
 [dependencies]
 num = "0.1"
 rand = "0.3"
 approx = "0.1"
-cgmath = "0.15"
+serde_derive = "1.0"
```

Comment:
But why not just put it below the cgmath line?

Reply:
I tried that first but it didn't work due to what I believe is a bug in cargo.

> Unfortunately my setup here doesn't seem to work with proc-macro crates such as serde_derive. Fortunately I think including serde_derive in the general dependency list shouldn't be a problem as serde_derive only consists of preprocessors, which have no impact on the compiled object if they simply aren't used.